tests/dataframe/test_joins.py

```diff
@@ -3,13 +3,25 @@
 import pyarrow as pa
 import pytest
 
+import daft
 from daft.datatype import DataType
 from daft.errors import ExpressionTypeError
 from tests.utils import sort_arrow_table
 
 
+@pytest.fixture(params=[False, True])
+def broadcast_join_enabled(request):
+    # Toggles between default broadcast join threshold (10 MiB), and a threshold of 0, which disables broadcast joins.
+    broadcast_threshold = 10 * 1024 * 1024 if request.param else False
```

Comment:
Did you mean `else None` here? Surprised type-checking didn't catch this

Reply:
Ah this was supposed to be `else 0`, and type checking didn't catch this since Python booleans are actually ints! 😄